src/xrCore/xr_cpuid.cpp

```diff
@@ -193,67 +259,71 @@ bool query_processor_info(processor_info* pinfo)
     pinfo->model = (cpui[0] >> 4) & 0xf;
     pinfo->stepping = cpui[0] & 0xf;
 
-    // Calculate available processors
-#ifdef XR_PLATFORM_WINDOWS
-    ULONG_PTR pa_mask_save, sa_mask_stub = 0;
-    GetProcessAffinityMask(GetCurrentProcess(), &pa_mask_save, &sa_mask_stub);
-#elif defined(XR_PLATFORM_LINUX) || defined(XR_PLATFORM_FREEBSD)
-    u32 pa_mask_save = 0;
-    cpu_set_t my_set;
-    CPU_ZERO(&my_set);
-    pthread_getaffinity_np(pthread_self(), sizeof(cpu_set_t), &my_set);
-    pa_mask_save = CPU_COUNT(&my_set);
-#else
-#pragma TODO("No function to obtain process affinity")
-    u32 pa_mask_save = 0;
-#endif // XR_PLATFORM_WINDOWS
+    fillInAvailableCpus(pinfo);
 
-    u32 processorCoreCount = 0;
-    u32 logicalProcessorCount = 0;
+    return pinfo->features.get() != 0;
+}
 
-#ifdef XR_PLATFORM_WINDOWS
-    DWORD returnedLength = 0;
-    GetLogicalProcessorInformation(nullptr, &returnedLength);
+#elif defined(XR_ARCHITECTURE_E2K)
+bool query_processor_info(processor_info* pinfo)
+{
+    *pinfo = {};
 
-    auto* buffer = (SYSTEM_LOGICAL_PROCESSOR_INFORMATION*)xr_alloca(returnedLength);
-    GetLogicalProcessorInformation(buffer, &returnedLength);
+    strcpy(pinfo->vendor, "MCST");
+    xr_sprintf(pinfo->modelName, "%s (%s)", __builtin_cpu_name(), __builtin_cpu_arch());
+    
 
-    u32 byteOffset = 0;
-    while (byteOffset + sizeof(SYSTEM_LOGICAL_PROCESSOR_INFORMATION) <= returnedLength)
-    {
-        switch (buffer->Relationship)
-        {
-            case RelationProcessorCore:
-                processorCoreCount++;
+    #if (defined(__MMX__))
```

Comment:
`#` should be always to the left. Other part of the macro can be aligned, though.
Also excessive parentheses are unneeded.
```suggestion
#if defined(__MMX__)
```

Reply:
Remove the alignment.

> Other part of the macro can be aligned, though.

By this sentence I mean alignment in nesting cases, like this:
```cpp
#if defined(DEFINE)
#   if defined(DEFINE2)
#       define ONE
#	else
#       define TWO
#	endif
#endif
```
But there's no nesting and `#` character should always stick to the left side.